src/RockPaperScissors.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RockPaperScissors.h"
+#include <string>
+using namespace std;
+
+string RockPaperScissorsPlay(const string player1, const string player2)
+{
+    if (player1 == "Scissors" && player2 == "Rock")
+    {
+        return "Player1 Loses";
```

Comment:
You've got the same value `"Player1 Loses"` repeated a lot of times in this file (same with `"Scissors"` / `"Rock"`). If someone is working on this code and makes a typo such as `"Player1 loses"`, that difference would be hard to track down.

Are there any ways you can think of to work around this?